### PR TITLE
Fix/windows tests

### PR DESCRIPTION
**Pull request by michaellilltokiwa**
This PR should fix the tests on windows.

left linux, right windows:

image

### CODE REVIEW

File: tests/_cur_dir.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (fridis) [inline tests/_cur_dir.sh:1]: please add the Fuzion header like in other scripts

**Review by fridis — CHANGES_REQUESTED** [review]
looks good.

File: tests/_cur_dir.sh (hunk 1 above)
Comment (michaellilltokiwa) [reply tests/_cur_dir.sh:26]: @fridis done

File: tests/_cur_dir.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#
+# You should have received a copy of the GNU General Public License along with The
+# Fuzion language implementation.  If not, see <https://www.gnu.org/licenses/>.
+
+
+# -----------------------------------------------------------------------
+#
+#  Tokiwa Software GmbH, Germany
+#
+#  Source code of fz command, the main Fuzion tools entry point
```
Comment (fridis) [inline tests/_cur_dir.sh:20]: this needs an update

File: tests/_cur_dir.sh [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+#
+#  Author: Fridtjof Siebert (siebert@tokiwa.software)
```
Comment (fridis) [inline tests/_cur_dir.sh:22]: and this should be you.

**Review by fridis — CHANGES_REQUESTED** [review]

**Pull request merged**